src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactReleaseHistoryScoreTest.java

```diff
@@ -98,8 +100,8 @@ public void testCurrentReleasedVersionBestHistory() {
     ArtifactReleaseHistoryScore score = new ArtifactReleaseHistoryScore();
     Value<ArtifactVersions> versions = RELEASED_ARTIFACT_VERSIONS.value(
         ArtifactVersions.of(version100, version110, version120));
-    ScoreValue value = score.calculate(versions);
-    Assert.assertEquals(9.5, value.get(), DELTA);
+    ScoreValue value = score.calculate(versions, ARTIFACT_VERSION.unknown());
```

Comment:
Why don't we use unknown artifact versions? I think it would be better to use real versions and add a few new test cases with unknown values.